[root@master ~]# kubectl --namespace kube-system logs kube-scheduler-master.k8s.com
I0119 17:17:30.244378       1 serving.go:331] Generated self-signed cert in-memory
W0119 17:17:31.415505       1 authentication.go:332] Error looking up in-cluster authentication configuration: Get "https://192.168.18.15:6443/api/v1/namespaces/kube-system/configmaps/extension-apiserver-authentication?timeout=10s": dial tcp 192.168.18.15:6443: connect: network is unreachable
W0119 17:17:31.415550       1 authentication.go:333] Continuing without authentication configuration. This may treat all requests as anonymous.
W0119 17:17:31.415559       1 authentication.go:334] To require authentication configuration lookup to succeed, set --authentication-tolerate-lookup-failure=false
I0119 17:17:31.442007       1 secure_serving.go:197] Serving securely on 127.0.0.1:10259
E0119 17:17:31.442897       1 reflector.go:138] k8s.io/client-go/informers/factory.go:134: Failed to watch *v1.CSINode: failed to list *v1.CSINode: Get "https://192.168.18.15:6443/apis/storage.k8s.io/v1/csinodes?limit=500&resourceVersion=0": dial tcp 192.168.18.15:6443: connect: network is unreachable
I0119 17:17:31.442976       1 configmap_cafile_content.go:202] Starting client-ca::kube-system::extension-apiserver-authentication::client-ca-file
I0119 17:17:31.442984       1 shared_informer.go:240] Waiting for caches to sync for client-ca::kube-system::extension-apiserver-authentication::client-ca-file
I0119 17:17:31.443000       1 tlsconfig.go:240] Starting DynamicServingCertificateController
E0119 17:17:31.444065       1 reflector.go:138] k8s.io/apiserver/pkg/server/dynamiccertificates/configmap_cafile_content.go:206: Failed to watch *v1.ConfigMap: failed to list *v1.ConfigMap: Get "https://192.168.18.15:6443/api/v1/namespaces/kube-system/configmaps?fieldSelector=metadata.name%3Dextension-apiserver-authentication&limit=500&resourceVersion=0": dial tcp 192.168.18.15:6443: connect: network is unreachable
E0119 17:17:31.447999       1 reflector.go:138] k8s.io/client-go/informers/factory.go:134: Failed to watch *v1.StorageClass: failed to list *v1.StorageClass: Get "https://192.168.18.15:6443/apis/storage.k8s.io/v1/storageclasses?limit=500&resourceVersion=0": dial tcp 192.168.18.15:6443: connect: network is unreachable
E0119 17:17:31.457149       1 reflector.go:138] k8s.io/client-go/informers/factory.go:134: Failed to watch *v1.Service: failed to list *v1.Service: Get "https://192.168.18.15:6443/api/v1/services?limit=500&resourceVersion=0": dial tcp 192.168.18.15:6443: connect: network is unreachable
E0119 17:17:31.457606       1 reflector.go:138] k8s.io/client-go/informers/factory.go:134: Failed to watch *v1.StatefulSet: failed to list *v1.StatefulSet: Get "https://192.168.18.15:6443/apis/apps/v1/statefulsets?limit=500&resourceVersion=0": dial tcp 192.168.18.15:6443: connect: network is unreachable
E0119 17:17:31.458046       1 reflector.go:138] k8s.io/client-go/informers/factory.go:134: Failed to watch *v1.Pod: failed to list *v1.Pod: Get "https://192.168.18.15:6443/api/v1/pods?fieldSelector=status.phase%21%3DSucceeded%2Cstatus.phase%21%3DFailed&limit=500&resourceVersion=0": dial tcp 192.168.18.15:6443: connect: network is unreachable
E0119 17:17:31.458609       1 reflector.go:138] k8s.io/client-go/informers/factory.go:134: Failed to watch *v1.ReplicationController: failed to list *v1.ReplicationController: Get "https://192.168.18.15:6443/api/v1/replicationcontrollers?limit=500&resourceVersion=0": dial tcp 192.168.18.15:6443: connect: network is unreachable
E0119 17:17:31.473708       1 reflector.go:138] k8s.io/client-go/informers/factory.go:134: Failed to watch *v1.PersistentVolumeClaim: failed to list *v1.PersistentVolumeClaim: Get "https://192.168.18.15:6443/api/v1/persistentvolumeclaims?limit=500&resourceVersion=0": dial tcp 192.168.18.15:6443: connect: network is unreachable
E0119 17:17:31.474184       1 reflector.go:138] k8s.io/client-go/informers/factory.go:134: Failed to watch *v1.ReplicaSet: failed to list *v1.ReplicaSet: Get "https://192.168.18.15:6443/apis/apps/v1/replicasets?limit=500&resourceVersion=0": dial tcp 192.168.18.15:6443: connect: network is unreachable
E0119 17:17:31.475227       1 reflector.go:138] k8s.io/client-go/informers/factory.go:134: Failed to watch *v1.PersistentVolume: failed to list *v1.PersistentVolume: Get "https://192.168.18.15:6443/api/v1/persistentvolumes?limit=500&resourceVersion=0": dial tcp 192.168.18.15:6443: connect: network is unreachable
E0119 17:17:31.480936       1 reflector.go:138] k8s.io/client-go/informers/factory.go:134: Failed to watch *v1beta1.PodDisruptionBudget: failed to list *v1beta1.PodDisruptionBudget: Get "https://192.168.18.15:6443/apis/policy/v1beta1/poddisruptionbudgets?limit=500&resourceVersion=0": dial tcp 192.168.18.15:6443: connect: network is unreachable
E0119 17:17:31.523858       1 reflector.go:138] k8s.io/client-go/informers/factory.go:134: Failed to watch *v1.Node: failed to list *v1.Node: Get "https://192.168.18.15:6443/api/v1/nodes?limit=500&resourceVersion=0": dial tcp 192.168.18.15:6443: connect: network is unreachable
E0119 17:17:32.322891       1 reflector.go:138] k8s.io/apiserver/pkg/server/dynamiccertificates/configmap_cafile_content.go:206: Failed to watch *v1.ConfigMap: failed to list *v1.ConfigMap: Get "https://192.168.18.15:6443/api/v1/namespaces/kube-system/configmaps?fieldSelector=metadata.name%3Dextension-apiserver-authentication&limit=500&resourceVersion=0": dial tcp 192.168.18.15:6443: connect: network is unreachable
E0119 17:17:32.334183       1 reflector.go:138] k8s.io/client-go/informers/factory.go:134: Failed to watch *v1beta1.PodDisruptionBudget: failed to list *v1beta1.PodDisruptionBudget: Get "https://192.168.18.15:6443/apis/policy/v1beta1/poddisruptionbudgets?limit=500&resourceVersion=0": dial tcp 192.168.18.15:6443: connect: network is unreachable
E0119 17:17:32.385997       1 reflector.go:138] k8s.io/client-go/informers/factory.go:134: Failed to watch *v1.StorageClass: failed to list *v1.StorageClass: Get "https://192.168.18.15:6443/apis/storage.k8s.io/v1/storageclasses?limit=500&resourceVersion=0": dial tcp 192.168.18.15:6443: connect: network is unreachable
E0119 17:17:32.421223       1 reflector.go:138] k8s.io/client-go/informers/factory.go:134: Failed to watch *v1.ReplicationController: failed to list *v1.ReplicationController: Get "https://192.168.18.15:6443/api/v1/replicationcontrollers?limit=500&resourceVersion=0": dial tcp 192.168.18.15:6443: connect: network is unreachable
E0119 17:17:32.637224       1 reflector.go:138] k8s.io/client-go/informers/factory.go:134: Failed to watch *v1.PersistentVolumeClaim: failed to list *v1.PersistentVolumeClaim: Get "https://192.168.18.15:6443/api/v1/persistentvolumeclaims?limit=500&resourceVersion=0": dial tcp 192.168.18.15:6443: connect: network is unreachable
E0119 17:17:32.674227       1 reflector.go:138] k8s.io/client-go/informers/factory.go:134: Failed to watch *v1.PersistentVolume: failed to list *v1.PersistentVolume: Get "https://192.168.18.15:6443/api/v1/persistentvolumes?limit=500&resourceVersion=0": dial tcp 192.168.18.15:6443: connect: network is unreachable
E0119 17:17:32.741886       1 reflector.go:138] k8s.io/client-go/informers/factory.go:134: Failed to watch *v1.Pod: failed to list *v1.Pod: Get "https://192.168.18.15:6443/api/v1/pods?fieldSelector=status.phase%21%3DSucceeded%2Cstatus.phase%21%3DFailed&limit=500&resourceVersion=0": dial tcp 192.168.18.15:6443: connect: network is unreachable
E0119 17:17:32.799476       1 reflector.go:138] k8s.io/client-go/informers/factory.go:134: Failed to watch *v1.StatefulSet: failed to list *v1.StatefulSet: Get "https://192.168.18.15:6443/apis/apps/v1/statefulsets?limit=500&resourceVersion=0": dial tcp 192.168.18.15:6443: connect: network is unreachable
E0119 17:17:32.846732       1 reflector.go:138] k8s.io/client-go/informers/factory.go:134: Failed to watch *v1.Service: failed to list *v1.Service: Get "https://192.168.18.15:6443/api/v1/services?limit=500&resourceVersion=0": dial tcp 192.168.18.15:6443: connect: network is unreachable
E0119 17:17:32.926782       1 reflector.go:138] k8s.io/client-go/informers/factory.go:134: Failed to watch *v1.Node: failed to list *v1.Node: Get "https://192.168.18.15:6443/api/v1/nodes?limit=500&resourceVersion=0": dial tcp 192.168.18.15:6443: connect: network is unreachable
E0119 17:17:32.956660       1 reflector.go:138] k8s.io/client-go/informers/factory.go:134: Failed to watch *v1.ReplicaSet: failed to list *v1.ReplicaSet: Get "https://192.168.18.15:6443/apis/apps/v1/replicasets?limit=500&resourceVersion=0": dial tcp 192.168.18.15:6443: connect: network is unreachable
E0119 17:17:32.981984       1 reflector.go:138] k8s.io/client-go/informers/factory.go:134: Failed to watch *v1.CSINode: failed to list *v1.CSINode: Get "https://192.168.18.15:6443/apis/storage.k8s.io/v1/csinodes?limit=500&resourceVersion=0": dial tcp 192.168.18.15:6443: connect: network is unreachable
E0119 17:17:34.189157       1 reflector.go:138] k8s.io/client-go/informers/factory.go:134: Failed to watch *v1beta1.PodDisruptionBudget: failed to list *v1beta1.PodDisruptionBudget: Get "https://192.168.18.15:6443/apis/policy/v1beta1/poddisruptionbudgets?limit=500&resourceVersion=0": dial tcp 192.168.18.15:6443: connect: network is unreachable
E0119 17:17:34.442178       1 reflector.go:138] k8s.io/apiserver/pkg/server/dynamiccertificates/configmap_cafile_content.go:206: Failed to watch *v1.ConfigMap: failed to list *v1.ConfigMap: Get "https://192.168.18.15:6443/api/v1/namespaces/kube-system/configmaps?fieldSelector=metadata.name%3Dextension-apiserver-authentication&limit=500&resourceVersion=0": dial tcp 192.168.18.15:6443: connect: network is unreachable
E0119 17:17:34.479349       1 reflector.go:138] k8s.io/client-go/informers/factory.go:134: Failed to watch *v1.StorageClass: failed to list *v1.StorageClass: Get "https://192.168.18.15:6443/apis/storage.k8s.io/v1/storageclasses?limit=500&resourceVersion=0": dial tcp 192.168.18.15:6443: connect: network is unreachable
E0119 17:17:34.678697       1 reflector.go:138] k8s.io/client-go/informers/factory.go:134: Failed to watch *v1.PersistentVolume: failed to list *v1.PersistentVolume: Get "https://192.168.18.15:6443/api/v1/persistentvolumes?limit=500&resourceVersion=0": dial tcp 192.168.18.15:6443: connect: network is unreachable
E0119 17:17:34.995359       1 reflector.go:138] k8s.io/client-go/informers/factory.go:134: Failed to watch *v1.StatefulSet: failed to list *v1.StatefulSet: Get "https://192.168.18.15:6443/apis/apps/v1/statefulsets?limit=500&resourceVersion=0": dial tcp 192.168.18.15:6443: connect: network is unreachable
E0119 17:17:35.089264       1 reflector.go:138] k8s.io/client-go/informers/factory.go:134: Failed to watch *v1.Pod: failed to list *v1.Pod: Get "https://192.168.18.15:6443/api/v1/pods?fieldSelector=status.phase%21%3DSucceeded%2Cstatus.phase%21%3DFailed&limit=500&resourceVersion=0": dial tcp 192.168.18.15:6443: connect: network is unreachable
E0119 17:17:35.207628       1 reflector.go:138] k8s.io/client-go/informers/factory.go:134: Failed to watch *v1.Node: failed to list *v1.Node: Get "https://192.168.18.15:6443/api/v1/nodes?limit=500&resourceVersion=0": dial tcp 192.168.18.15:6443: connect: network is unreachable
E0119 17:17:35.262846       1 reflector.go:138] k8s.io/client-go/informers/factory.go:134: Failed to watch *v1.PersistentVolumeClaim: failed to list *v1.PersistentVolumeClaim: Get "https://192.168.18.15:6443/api/v1/persistentvolumeclaims?limit=500&resourceVersion=0": dial tcp 192.168.18.15:6443: connect: network is unreachable
E0119 17:17:35.341411       1 reflector.go:138] k8s.io/client-go/informers/factory.go:134: Failed to watch *v1.ReplicationController: failed to list *v1.ReplicationController: Get "https://192.168.18.15:6443/api/v1/replicationcontrollers?limit=500&resourceVersion=0": dial tcp 192.168.18.15:6443: connect: network is unreachable
E0119 17:17:35.431181       1 reflector.go:138] k8s.io/client-go/informers/factory.go:134: Failed to watch *v1.CSINode: failed to list *v1.CSINode: Get "https://192.168.18.15:6443/apis/storage.k8s.io/v1/csinodes?limit=500&resourceVersion=0": dial tcp 192.168.18.15:6443: connect: network is unreachable
E0119 17:17:35.480037       1 reflector.go:138] k8s.io/client-go/informers/factory.go:134: Failed to watch *v1.Service: failed to list *v1.Service: Get "https://192.168.18.15:6443/api/v1/services?limit=500&resourceVersion=0": dial tcp 192.168.18.15:6443: connect: network is unreachable
E0119 17:17:36.060074       1 reflector.go:138] k8s.io/client-go/informers/factory.go:134: Failed to watch *v1.ReplicaSet: failed to list *v1.ReplicaSet: Get "https://192.168.18.15:6443/apis/apps/v1/replicasets?limit=500&resourceVersion=0": dial tcp 192.168.18.15:6443: connect: network is unreachable
E0119 17:17:38.306861       1 reflector.go:138] k8s.io/apiserver/pkg/server/dynamiccertificates/configmap_cafile_content.go:206: Failed to watch *v1.ConfigMap: failed to list *v1.ConfigMap: Get "https://192.168.18.15:6443/api/v1/namespaces/kube-system/configmaps?fieldSelector=metadata.name%3Dextension-apiserver-authentication&limit=500&resourceVersion=0": dial tcp 192.168.18.15:6443: connect: network is unreachable
E0119 17:17:38.318576       1 reflector.go:138] k8s.io/client-go/informers/factory.go:134: Failed to watch *v1.StorageClass: failed to list *v1.StorageClass: Get "https://192.168.18.15:6443/apis/storage.k8s.io/v1/storageclasses?limit=500&resourceVersion=0": dial tcp 192.168.18.15:6443: connect: network is unreachable
E0119 17:17:38.924474       1 reflector.go:138] k8s.io/client-go/informers/factory.go:134: Failed to watch *v1.StatefulSet: failed to list *v1.StatefulSet: Get "https://192.168.18.15:6443/apis/apps/v1/statefulsets?limit=500&resourceVersion=0": dial tcp 192.168.18.15:6443: connect: network is unreachable
E0119 17:17:39.229249       1 reflector.go:138] k8s.io/client-go/informers/factory.go:134: Failed to watch *v1beta1.PodDisruptionBudget: failed to list *v1beta1.PodDisruptionBudget: Get "https://192.168.18.15:6443/apis/policy/v1beta1/poddisruptionbudgets?limit=500&resourceVersion=0": dial tcp 192.168.18.15:6443: connect: network is unreachable
E0119 17:17:39.560514       1 reflector.go:138] k8s.io/client-go/informers/factory.go:134: Failed to watch *v1.Service: failed to list *v1.Service: Get "https://192.168.18.15:6443/api/v1/services?limit=500&resourceVersion=0": dial tcp 192.168.18.15:6443: connect: network is unreachable
E0119 17:17:39.841141       1 reflector.go:138] k8s.io/client-go/informers/factory.go:134: Failed to watch *v1.Node: failed to list *v1.Node: Get "https://192.168.18.15:6443/api/v1/nodes?limit=500&resourceVersion=0": dial tcp 192.168.18.15:6443: connect: network is unreachable
E0119 17:17:39.861523       1 reflector.go:138] k8s.io/client-go/informers/factory.go:134: Failed to watch *v1.ReplicationController: failed to list *v1.ReplicationController: Get "https://192.168.18.15:6443/api/v1/replicationcontrollers?limit=500&resourceVersion=0": dial tcp 192.168.18.15:6443: connect: network is unreachable
E0119 17:17:39.882006       1 reflector.go:138] k8s.io/client-go/informers/factory.go:134: Failed to watch *v1.PersistentVolumeClaim: failed to list *v1.PersistentVolumeClaim: Get "https://192.168.18.15:6443/api/v1/persistentvolumeclaims?limit=500&resourceVersion=0": dial tcp 192.168.18.15:6443: connect: network is unreachable
E0119 17:17:39.975048       1 reflector.go:138] k8s.io/client-go/informers/factory.go:134: Failed to watch *v1.CSINode: failed to list *v1.CSINode: Get "https://192.168.18.15:6443/apis/storage.k8s.io/v1/csinodes?limit=500&resourceVersion=0": dial tcp 192.168.18.15:6443: connect: network is unreachable
E0119 17:17:40.021852       1 reflector.go:138] k8s.io/client-go/informers/factory.go:134: Failed to watch *v1.Pod: failed to list *v1.Pod: Get "https://192.168.18.15:6443/api/v1/pods?fieldSelector=status.phase%21%3DSucceeded%2Cstatus.phase%21%3DFailed&limit=500&resourceVersion=0": dial tcp 192.168.18.15:6443: connect: network is unreachable
E0119 17:17:40.893327       1 reflector.go:138] k8s.io/client-go/informers/factory.go:134: Failed to watch *v1.PersistentVolume: failed to list *v1.PersistentVolume: Get "https://192.168.18.15:6443/api/v1/persistentvolumes?limit=500&resourceVersion=0": dial tcp 192.168.18.15:6443: connect: network is unreachable
E0119 17:17:41.937003       1 reflector.go:138] k8s.io/client-go/informers/factory.go:134: Failed to watch *v1.ReplicaSet: failed to list *v1.ReplicaSet: Get "https://192.168.18.15:6443/apis/apps/v1/replicasets?limit=500&resourceVersion=0": dial tcp 192.168.18.15:6443: connect: network is unreachable
E0119 17:17:45.173680       1 reflector.go:138] k8s.io/client-go/informers/factory.go:134: Failed to watch *v1.StorageClass: failed to list *v1.StorageClass: Get "https://192.168.18.15:6443/apis/storage.k8s.io/v1/storageclasses?limit=500&resourceVersion=0": dial tcp 192.168.18.15:6443: connect: network is unreachable
E0119 17:17:46.445488       1 reflector.go:138] k8s.io/client-go/informers/factory.go:134: Failed to watch *v1.StatefulSet: failed to list *v1.StatefulSet: Get "https://192.168.18.15:6443/apis/apps/v1/statefulsets?limit=500&resourceVersion=0": dial tcp 192.168.18.15:6443: connect: network is unreachable
E0119 17:17:46.674680       1 reflector.go:138] k8s.io/client-go/informers/factory.go:134: Failed to watch *v1.Pod: failed to list *v1.Pod: Get "https://192.168.18.15:6443/api/v1/pods?fieldSelector=status.phase%21%3DSucceeded%2Cstatus.phase%21%3DFailed&limit=500&resourceVersion=0": dial tcp 192.168.18.15:6443: connect: network is unreachable
E0119 17:17:47.954749       1 reflector.go:138] k8s.io/client-go/informers/factory.go:134: Failed to watch *v1beta1.PodDisruptionBudget: failed to list *v1beta1.PodDisruptionBudget: Get "https://192.168.18.15:6443/apis/policy/v1beta1/poddisruptionbudgets?limit=500&resourceVersion=0": dial tcp 192.168.18.15:6443: connect: network is unreachable
E0119 17:17:48.394886       1 reflector.go:138] k8s.io/client-go/informers/factory.go:134: Failed to watch *v1.PersistentVolume: failed to list *v1.PersistentVolume: Get "https://192.168.18.15:6443/api/v1/persistentvolumes?limit=500&resourceVersion=0": dial tcp 192.168.18.15:6443: connect: network is unreachable
E0119 17:17:49.926556       1 reflector.go:138] k8s.io/client-go/informers/factory.go:134: Failed to watch *v1.Node: failed to list *v1.Node: Get "https://192.168.18.15:6443/api/v1/nodes?limit=500&resourceVersion=0": dial tcp 192.168.18.15:6443: connect: network is unreachable
E0119 17:17:50.473043       1 reflector.go:138] k8s.io/apiserver/pkg/server/dynamiccertificates/configmap_cafile_content.go:206: Failed to watch *v1.ConfigMap: failed to list *v1.ConfigMap: Get "https://192.168.18.15:6443/api/v1/namespaces/kube-system/configmaps?fieldSelector=metadata.name%3Dextension-apiserver-authentication&limit=500&resourceVersion=0": dial tcp 192.168.18.15:6443: connect: network is unreachable
E0119 17:17:50.621028       1 reflector.go:138] k8s.io/client-go/informers/factory.go:134: Failed to watch *v1.ReplicationController: failed to list *v1.ReplicationController: Get "https://192.168.18.15:6443/api/v1/replicationcontrollers?limit=500&resourceVersion=0": dial tcp 192.168.18.15:6443: connect: network is unreachable
E0119 17:17:50.825931       1 reflector.go:138] k8s.io/client-go/informers/factory.go:134: Failed to watch *v1.Service: failed to list *v1.Service: Get "https://192.168.18.15:6443/api/v1/services?limit=500&resourceVersion=0": dial tcp 192.168.18.15:6443: connect: network is unreachable
E0119 17:17:51.232580       1 reflector.go:138] k8s.io/client-go/informers/factory.go:134: Failed to watch *v1.PersistentVolumeClaim: failed to list *v1.PersistentVolumeClaim: Get "https://192.168.18.15:6443/api/v1/persistentvolumeclaims?limit=500&resourceVersion=0": dial tcp 192.168.18.15:6443: connect: network is unreachable
E0119 17:17:52.164932       1 reflector.go:138] k8s.io/client-go/informers/factory.go:134: Failed to watch *v1.CSINode: failed to list *v1.CSINode: Get "https://192.168.18.15:6443/apis/storage.k8s.io/v1/csinodes?limit=500&resourceVersion=0": dial tcp 192.168.18.15:6443: connect: network is unreachable
E0119 17:17:53.641105       1 reflector.go:138] k8s.io/client-go/informers/factory.go:134: Failed to watch *v1.ReplicaSet: failed to list *v1.ReplicaSet: Get "https://192.168.18.15:6443/apis/apps/v1/replicasets?limit=500&resourceVersion=0": dial tcp 192.168.18.15:6443: connect: network is unreachable
E0119 17:17:59.485329       1 reflector.go:138] k8s.io/client-go/informers/factory.go:134: Failed to watch *v1.StatefulSet: failed to list *v1.StatefulSet: Get "https://192.168.18.15:6443/apis/apps/v1/statefulsets?limit=500&resourceVersion=0": dial tcp 192.168.18.15:6443: connect: network is unreachable
E0119 17:18:03.878918       1 reflector.go:138] k8s.io/client-go/informers/factory.go:134: Failed to watch *v1.PersistentVolume: failed to list *v1.PersistentVolume: Get "https://192.168.18.15:6443/api/v1/persistentvolumes?limit=500&resourceVersion=0": dial tcp 192.168.18.15:6443: connect: network is unreachable
E0119 17:18:04.217235       1 reflector.go:138] k8s.io/client-go/informers/factory.go:134: Failed to watch *v1.Pod: failed to list *v1.Pod: Get "https://192.168.18.15:6443/api/v1/pods?fieldSelector=status.phase%21%3DSucceeded%2Cstatus.phase%21%3DFailed&limit=500&resourceVersion=0": dial tcp 192.168.18.15:6443: connect: network is unreachable
E0119 17:18:04.780309       1 reflector.go:138] k8s.io/client-go/informers/factory.go:134: Failed to watch *v1.Service: failed to list *v1.Service: Get "https://192.168.18.15:6443/api/v1/services?limit=500&resourceVersion=0": dial tcp 192.168.18.15:6443: connect: network is unreachable
E0119 17:18:05.074519       1 reflector.go:138] k8s.io/client-go/informers/factory.go:134: Failed to watch *v1.Node: failed to list *v1.Node: Get "https://192.168.18.15:6443/api/v1/nodes?limit=500&resourceVersion=0": dial tcp 192.168.18.15:6443: connect: network is unreachable
E0119 17:18:10.000901       1 reflector.go:138] k8s.io/client-go/informers/factory.go:134: Failed to watch *v1beta1.PodDisruptionBudget: failed to list *v1beta1.PodDisruptionBudget: Get "https://192.168.18.15:6443/apis/policy/v1beta1/poddisruptionbudgets?limit=500&resourceVersion=0": dial tcp 192.168.18.15:6443: connect: network is unreachable
E0119 17:18:10.163551       1 reflector.go:138] k8s.io/client-go/informers/factory.go:134: Failed to watch *v1.StorageClass: failed to list *v1.StorageClass: Get "https://192.168.18.15:6443/apis/storage.k8s.io/v1/storageclasses?limit=500&resourceVersion=0": dial tcp 192.168.18.15:6443: connect: network is unreachable
E0119 17:18:11.801422       1 reflector.go:138] k8s.io/apiserver/pkg/server/dynamiccertificates/configmap_cafile_content.go:206: Failed to watch *v1.ConfigMap: failed to list *v1.ConfigMap: Get "https://192.168.18.15:6443/api/v1/namespaces/kube-system/configmaps?fieldSelector=metadata.name%3Dextension-apiserver-authentication&limit=500&resourceVersion=0": dial tcp 192.168.18.15:6443: connect: network is unreachable
E0119 17:18:12.032571       1 reflector.go:138] k8s.io/client-go/informers/factory.go:134: Failed to watch *v1.ReplicationController: failed to list *v1.ReplicationController: Get "https://192.168.18.15:6443/api/v1/replicationcontrollers?limit=500&resourceVersion=0": dial tcp 192.168.18.15:6443: connect: network is unreachable
E0119 17:18:13.383973       1 reflector.go:138] k8s.io/client-go/informers/factory.go:134: Failed to watch *v1.ReplicaSet: failed to list *v1.ReplicaSet: Get "https://192.168.18.15:6443/apis/apps/v1/replicasets?limit=500&resourceVersion=0": dial tcp 192.168.18.15:6443: connect: network is unreachable
E0119 17:18:14.874070       1 reflector.go:138] k8s.io/client-go/informers/factory.go:134: Failed to watch *v1.PersistentVolumeClaim: failed to list *v1.PersistentVolumeClaim: Get "https://192.168.18.15:6443/api/v1/persistentvolumeclaims?limit=500&resourceVersion=0": dial tcp 192.168.18.15:6443: connect: network is unreachable
E0119 17:18:15.574259       1 reflector.go:138] k8s.io/client-go/informers/factory.go:134: Failed to watch *v1.CSINode: failed to list *v1.CSINode: Get "https://192.168.18.15:6443/apis/storage.k8s.io/v1/csinodes?limit=500&resourceVersion=0": dial tcp 192.168.18.15:6443: connect: network is unreachable
E0119 17:18:34.318651       1 reflector.go:138] k8s.io/client-go/informers/factory.go:134: Failed to watch *v1.PersistentVolume: failed to list *v1.PersistentVolume: Get "https://192.168.18.15:6443/api/v1/persistentvolumes?limit=500&resourceVersion=0": dial tcp 192.168.18.15:6443: connect: network is unreachable
E0119 17:18:36.059862       1 reflector.go:138] k8s.io/client-go/informers/factory.go:134: Failed to watch *v1.Pod: failed to list *v1.Pod: Get "https://192.168.18.15:6443/api/v1/pods?fieldSelector=status.phase%21%3DSucceeded%2Cstatus.phase%21%3DFailed&limit=500&resourceVersion=0": dial tcp 192.168.18.15:6443: connect: network is unreachable
E0119 17:18:37.863378       1 reflector.go:138] k8s.io/client-go/informers/factory.go:134: Failed to watch *v1.Node: failed to list *v1.Node: Get "https://192.168.18.15:6443/api/v1/nodes?limit=500&resourceVersion=0": dial tcp 192.168.18.15:6443: connect: network is unreachable
E0119 17:18:42.487426       1 reflector.go:138] k8s.io/client-go/informers/factory.go:134: Failed to watch *v1.Service: failed to list *v1.Service: Get "https://192.168.18.15:6443/api/v1/services?limit=500&resourceVersion=0": dial tcp 192.168.18.15:6443: connect: network is unreachable
E0119 17:18:43.315042       1 reflector.go:138] k8s.io/client-go/informers/factory.go:134: Failed to watch *v1.ReplicationController: failed to list *v1.ReplicationController: Get "https://192.168.18.15:6443/api/v1/replicationcontrollers?limit=500&resourceVersion=0": dial tcp 192.168.18.15:6443: connect: network is unreachable
E0119 17:18:43.498504       1 reflector.go:138] k8s.io/client-go/informers/factory.go:134: Failed to watch *v1.PersistentVolumeClaim: failed to list *v1.PersistentVolumeClaim: Get "https://192.168.18.15:6443/api/v1/persistentvolumeclaims?limit=500&resourceVersion=0": dial tcp 192.168.18.15:6443: connect: network is unreachable
E0119 17:18:45.288796       1 reflector.go:138] k8s.io/client-go/informers/factory.go:134: Failed to watch *v1.StatefulSet: failed to list *v1.StatefulSet: Get "https://192.168.18.15:6443/apis/apps/v1/statefulsets?limit=500&resourceVersion=0": dial tcp 192.168.18.15:6443: connect: network is unreachable
E0119 17:18:48.854485       1 reflector.go:138] k8s.io/client-go/informers/factory.go:134: Failed to watch *v1.ReplicaSet: failed to list *v1.ReplicaSet: Get "https://192.168.18.15:6443/apis/apps/v1/replicasets?limit=500&resourceVersion=0": dial tcp 192.168.18.15:6443: connect: network is unreachable
E0119 17:18:49.073798       1 reflector.go:138] k8s.io/apiserver/pkg/server/dynamiccertificates/configmap_cafile_content.go:206: Failed to watch *v1.ConfigMap: failed to list *v1.ConfigMap: Get "https://192.168.18.15:6443/api/v1/namespaces/kube-system/configmaps?fieldSelector=metadata.name%3Dextension-apiserver-authentication&limit=500&resourceVersion=0": dial tcp 192.168.18.15:6443: connect: network is unreachable
E0119 17:18:52.477400       1 reflector.go:138] k8s.io/client-go/informers/factory.go:134: Failed to watch *v1.StorageClass: failed to list *v1.StorageClass: Get "https://192.168.18.15:6443/apis/storage.k8s.io/v1/storageclasses?limit=500&resourceVersion=0": dial tcp 192.168.18.15:6443: connect: network is unreachable
E0119 17:18:56.185466       1 reflector.go:138] k8s.io/client-go/informers/factory.go:134: Failed to watch *v1beta1.PodDisruptionBudget: failed to list *v1beta1.PodDisruptionBudget: Get "https://192.168.18.15:6443/apis/policy/v1beta1/poddisruptionbudgets?limit=500&resourceVersion=0": dial tcp 192.168.18.15:6443: connect: network is unreachable
E0119 17:19:00.861543       1 reflector.go:138] k8s.io/client-go/informers/factory.go:134: Failed to watch *v1.CSINode: failed to list *v1.CSINode: Get "https://192.168.18.15:6443/apis/storage.k8s.io/v1/csinodes?limit=500&resourceVersion=0": dial tcp 192.168.18.15:6443: connect: network is unreachable
E0119 17:19:14.346362       1 reflector.go:138] k8s.io/client-go/informers/factory.go:134: Failed to watch *v1.PersistentVolume: failed to list *v1.PersistentVolume: Get "https://192.168.18.15:6443/api/v1/persistentvolumes?limit=500&resourceVersion=0": dial tcp 192.168.18.15:6443: connect: network is unreachable
E0119 17:19:14.782116       1 reflector.go:138] k8s.io/client-go/informers/factory.go:134: Failed to watch *v1.Pod: failed to list *v1.Pod: Get "https://192.168.18.15:6443/api/v1/pods?fieldSelector=status.phase%21%3DSucceeded%2Cstatus.phase%21%3DFailed&limit=500&resourceVersion=0": dial tcp 192.168.18.15:6443: connect: network is unreachable
E0119 17:19:18.291023       1 reflector.go:138] k8s.io/client-go/informers/factory.go:134: Failed to watch *v1.Service: failed to list *v1.Service: Get "https://192.168.18.15:6443/api/v1/services?limit=500&resourceVersion=0": dial tcp 192.168.18.15:6443: connect: network is unreachable
E0119 17:19:20.480582       1 reflector.go:138] k8s.io/client-go/informers/factory.go:134: Failed to watch *v1.Node: failed to list *v1.Node: Get "https://192.168.18.15:6443/api/v1/nodes?limit=500&resourceVersion=0": dial tcp 192.168.18.15:6443: connect: network is unreachable
E0119 17:19:32.406117       1 reflector.go:138] k8s.io/client-go/informers/factory.go:134: Failed to watch *v1beta1.PodDisruptionBudget: failed to list *v1beta1.PodDisruptionBudget: Get "https://192.168.18.15:6443/apis/policy/v1beta1/poddisruptionbudgets?limit=500&resourceVersion=0": dial tcp 192.168.18.15:6443: connect: network is unreachable
E0119 17:19:35.977153       1 reflector.go:138] k8s.io/client-go/informers/factory.go:134: Failed to watch *v1.ReplicationController: failed to list *v1.ReplicationController: Get "https://192.168.18.15:6443/api/v1/replicationcontrollers?limit=500&resourceVersion=0": dial tcp 192.168.18.15:6443: connect: network is unreachable
E0119 17:19:38.022319       1 reflector.go:138] k8s.io/client-go/informers/factory.go:134: Failed to watch *v1.PersistentVolumeClaim: failed to list *v1.PersistentVolumeClaim: Get "https://192.168.18.15:6443/api/v1/persistentvolumeclaims?limit=500&resourceVersion=0": dial tcp 192.168.18.15:6443: connect: network is unreachable
E0119 17:19:38.588437       1 reflector.go:138] k8s.io/apiserver/pkg/server/dynamiccertificates/configmap_cafile_content.go:206: Failed to watch *v1.ConfigMap: failed to list *v1.ConfigMap: Get "https://192.168.18.15:6443/api/v1/namespaces/kube-system/configmaps?fieldSelector=metadata.name%3Dextension-apiserver-authentication&limit=500&resourceVersion=0": dial tcp 192.168.18.15:6443: connect: network is unreachable
E0119 17:19:41.843312       1 reflector.go:138] k8s.io/client-go/informers/factory.go:134: Failed to watch *v1.StatefulSet: failed to list *v1.StatefulSet: Get "https://192.168.18.15:6443/apis/apps/v1/statefulsets?limit=500&resourceVersion=0": dial tcp 192.168.18.15:6443: connect: network is unreachable
E0119 17:19:44.914384       1 reflector.go:138] k8s.io/client-go/informers/factory.go:134: Failed to watch *v1.CSINode: failed to list *v1.CSINode: Get "https://192.168.18.15:6443/apis/storage.k8s.io/v1/csinodes?limit=500&resourceVersion=0": dial tcp 192.168.18.15:6443: connect: network is unreachable
E0119 17:19:45.815147       1 reflector.go:138] k8s.io/client-go/informers/factory.go:134: Failed to watch *v1.Pod: failed to list *v1.Pod: Get "https://192.168.18.15:6443/api/v1/pods?fieldSelector=status.phase%21%3DSucceeded%2Cstatus.phase%21%3DFailed&limit=500&resourceVersion=0": dial tcp 192.168.18.15:6443: connect: network is unreachable
E0119 17:19:47.284261       1 reflector.go:138] k8s.io/client-go/informers/factory.go:134: Failed to watch *v1.ReplicaSet: failed to list *v1.ReplicaSet: Get "https://192.168.18.15:6443/apis/apps/v1/replicasets?limit=500&resourceVersion=0": dial tcp 192.168.18.15:6443: connect: network is unreachable
E0119 17:19:50.201538       1 reflector.go:138] k8s.io/client-go/informers/factory.go:134: Failed to watch *v1.StorageClass: failed to list *v1.StorageClass: Get "https://192.168.18.15:6443/apis/storage.k8s.io/v1/storageclasses?limit=500&resourceVersion=0": dial tcp 192.168.18.15:6443: connect: network is unreachable
E0119 17:19:55.444395       1 reflector.go:138] k8s.io/client-go/informers/factory.go:134: Failed to watch *v1.Node: failed to list *v1.Node: Get "https://192.168.18.15:6443/api/v1/nodes?limit=500&resourceVersion=0": dial tcp 192.168.18.15:6443: connect: network is unreachable
E0119 17:20:07.337569       1 reflector.go:138] k8s.io/client-go/informers/factory.go:134: Failed to watch *v1.Service: failed to list *v1.Service: Get "https://192.168.18.15:6443/api/v1/services?limit=500&resourceVersion=0": dial tcp 192.168.18.15:6443: connect: network is unreachable
E0119 17:20:08.993292       1 reflector.go:138] k8s.io/apiserver/pkg/server/dynamiccertificates/configmap_cafile_content.go:206: Failed to watch *v1.ConfigMap: failed to list *v1.ConfigMap: Get "https://192.168.18.15:6443/api/v1/namespaces/kube-system/configmaps?fieldSelector=metadata.name%3Dextension-apiserver-authentication&limit=500&resourceVersion=0": dial tcp 192.168.18.15:6443: connect: network is unreachable
E0119 17:20:13.915133       1 reflector.go:138] k8s.io/client-go/informers/factory.go:134: Failed to watch *v1.PersistentVolume: failed to list *v1.PersistentVolume: Get "https://192.168.18.15:6443/api/v1/persistentvolumes?limit=500&resourceVersion=0": dial tcp 192.168.18.15:6443: connect: network is unreachable
E0119 17:20:15.242749       1 reflector.go:138] k8s.io/client-go/informers/factory.go:134: Failed to watch *v1.ReplicationController: failed to list *v1.ReplicationController: Get "https://192.168.18.15:6443/api/v1/replicationcontrollers?limit=500&resourceVersion=0": dial tcp 192.168.18.15:6443: connect: network is unreachable
E0119 17:20:20.769261       1 reflector.go:138] k8s.io/client-go/informers/factory.go:134: Failed to watch *v1.CSINode: failed to list *v1.CSINode: Get "https://192.168.18.15:6443/apis/storage.k8s.io/v1/csinodes?limit=500&resourceVersion=0": dial tcp 192.168.18.15:6443: connect: network is unreachable
E0119 17:20:22.285414       1 reflector.go:138] k8s.io/client-go/informers/factory.go:134: Failed to watch *v1beta1.PodDisruptionBudget: failed to list *v1beta1.PodDisruptionBudget: Get "https://192.168.18.15:6443/apis/policy/v1beta1/poddisruptionbudgets?limit=500&resourceVersion=0": dial tcp 192.168.18.15:6443: connect: network is unreachable
E0119 17:20:22.647034       1 reflector.go:138] k8s.io/client-go/informers/factory.go:134: Failed to watch *v1.ReplicaSet: failed to list *v1.ReplicaSet: Get "https://192.168.18.15:6443/apis/apps/v1/replicasets?limit=500&resourceVersion=0": dial tcp 192.168.18.15:6443: connect: network is unreachable
E0119 17:20:23.221923       1 reflector.go:138] k8s.io/client-go/informers/factory.go:134: Failed to watch *v1.Pod: failed to list *v1.Pod: Get "https://192.168.18.15:6443/api/v1/pods?fieldSelector=status.phase%21%3DSucceeded%2Cstatus.phase%21%3DFailed&limit=500&resourceVersion=0": dial tcp 192.168.18.15:6443: connect: network is unreachable
E0119 17:20:29.168059       1 reflector.go:138] k8s.io/client-go/informers/factory.go:134: Failed to watch *v1.PersistentVolumeClaim: failed to list *v1.PersistentVolumeClaim: Get "https://192.168.18.15:6443/api/v1/persistentvolumeclaims?limit=500&resourceVersion=0": dial tcp 192.168.18.15:6443: connect: network is unreachable
E0119 17:20:30.181901       1 reflector.go:138] k8s.io/client-go/informers/factory.go:134: Failed to watch *v1.StatefulSet: failed to list *v1.StatefulSet: Get "https://192.168.18.15:6443/apis/apps/v1/statefulsets?limit=500&resourceVersion=0": dial tcp 192.168.18.15:6443: connect: network is unreachable
E0119 17:20:34.983819       1 reflector.go:138] k8s.io/client-go/informers/factory.go:134: Failed to watch *v1.StorageClass: failed to list *v1.StorageClass: Get "https://192.168.18.15:6443/apis/storage.k8s.io/v1/storageclasses?limit=500&resourceVersion=0": dial tcp 192.168.18.15:6443: connect: network is unreachable
E0119 17:20:47.376024       1 reflector.go:138] k8s.io/client-go/informers/factory.go:134: Failed to watch *v1.Service: failed to list *v1.Service: Get "https://192.168.18.15:6443/api/v1/services?limit=500&resourceVersion=0": dial tcp 192.168.18.15:6443: connect: network is unreachable
E0119 17:20:48.533906       1 reflector.go:138] k8s.io/client-go/informers/factory.go:134: Failed to watch *v1.Node: failed to list *v1.Node: Get "https://192.168.18.15:6443/api/v1/nodes?limit=500&resourceVersion=0": dial tcp 192.168.18.15:6443: connect: network is unreachable
E0119 17:20:48.708419       1 reflector.go:138] k8s.io/apiserver/pkg/server/dynamiccertificates/configmap_cafile_content.go:206: Failed to watch *v1.ConfigMap: failed to list *v1.ConfigMap: Get "https://192.168.18.15:6443/api/v1/namespaces/kube-system/configmaps?fieldSelector=metadata.name%3Dextension-apiserver-authentication&limit=500&resourceVersion=0": dial tcp 192.168.18.15:6443: connect: network is unreachable
E0119 17:20:54.560618       1 reflector.go:138] k8s.io/client-go/informers/factory.go:134: Failed to watch *v1beta1.PodDisruptionBudget: failed to list *v1beta1.PodDisruptionBudget: Get "https://192.168.18.15:6443/apis/policy/v1beta1/poddisruptionbudgets?limit=500&resourceVersion=0": dial tcp 192.168.18.15:6443: connect: network is unreachable
E0119 17:20:57.238118       1 reflector.go:138] k8s.io/client-go/informers/factory.go:134: Failed to watch *v1.PersistentVolume: failed to list *v1.PersistentVolume: Get "https://192.168.18.15:6443/api/v1/persistentvolumes?limit=500&resourceVersion=0": dial tcp 192.168.18.15:6443: connect: network is unreachable
E0119 17:21:00.550018       1 reflector.go:138] k8s.io/client-go/informers/factory.go:134: Failed to watch *v1.Pod: failed to list *v1.Pod: Get "https://192.168.18.15:6443/api/v1/pods?fieldSelector=status.phase%21%3DSucceeded%2Cstatus.phase%21%3DFailed&limit=500&resourceVersion=0": dial tcp 192.168.18.15:6443: connect: network is unreachable
E0119 17:21:01.276917       1 reflector.go:138] k8s.io/client-go/informers/factory.go:134: Failed to watch *v1.ReplicaSet: failed to list *v1.ReplicaSet: Get "https://192.168.18.15:6443/apis/apps/v1/replicasets?limit=500&resourceVersion=0": dial tcp 192.168.18.15:6443: connect: network is unreachable
E0119 17:21:03.590076       1 reflector.go:138] k8s.io/client-go/informers/factory.go:134: Failed to watch *v1.StatefulSet: failed to list *v1.StatefulSet: Get "https://192.168.18.15:6443/apis/apps/v1/statefulsets?limit=500&resourceVersion=0": dial tcp 192.168.18.15:6443: connect: network is unreachable
E0119 17:21:04.266299       1 reflector.go:138] k8s.io/client-go/informers/factory.go:134: Failed to watch *v1.PersistentVolumeClaim: failed to list *v1.PersistentVolumeClaim: Get "https://192.168.18.15:6443/api/v1/persistentvolumeclaims?limit=500&resourceVersion=0": dial tcp 192.168.18.15:6443: connect: network is unreachable
E0119 17:21:05.301571       1 reflector.go:138] k8s.io/client-go/informers/factory.go:134: Failed to watch *v1.ReplicationController: failed to list *v1.ReplicationController: Get "https://192.168.18.15:6443/api/v1/replicationcontrollers?limit=500&resourceVersion=0": dial tcp 192.168.18.15:6443: connect: network is unreachable
E0119 17:21:18.713815       1 reflector.go:138] k8s.io/client-go/informers/factory.go:134: Failed to watch *v1.CSINode: failed to list *v1.CSINode: Get "https://192.168.18.15:6443/apis/storage.k8s.io/v1/csinodes?limit=500&resourceVersion=0": dial tcp 192.168.18.15:6443: connect: network is unreachable
E0119 17:21:29.574783       1 reflector.go:138] k8s.io/client-go/informers/factory.go:134: Failed to watch *v1.Node: failed to list *v1.Node: Get "https://192.168.18.15:6443/api/v1/nodes?limit=500&resourceVersion=0": dial tcp 192.168.18.15:6443: connect: network is unreachable
E0119 17:21:34.164282       1 reflector.go:138] k8s.io/client-go/informers/factory.go:134: Failed to watch *v1.StorageClass: failed to list *v1.StorageClass: Get "https://192.168.18.15:6443/apis/storage.k8s.io/v1/storageclasses?limit=500&resourceVersion=0": dial tcp 192.168.18.15:6443: connect: network is unreachable
E0119 17:21:34.299546       1 reflector.go:138] k8s.io/client-go/informers/factory.go:134: Failed to watch *v1.StatefulSet: failed to list *v1.StatefulSet: Get "https://192.168.18.15:6443/apis/apps/v1/statefulsets?limit=500&resourceVersion=0": dial tcp 192.168.18.15:6443: connect: network is unreachable
E0119 17:21:40.128817       1 reflector.go:138] k8s.io/client-go/informers/factory.go:134: Failed to watch *v1beta1.PodDisruptionBudget: failed to list *v1beta1.PodDisruptionBudget: Get "https://192.168.18.15:6443/apis/policy/v1beta1/poddisruptionbudgets?limit=500&resourceVersion=0": dial tcp 192.168.18.15:6443: connect: network is unreachable
E0119 17:21:41.166355       1 reflector.go:138] k8s.io/client-go/informers/factory.go:134: Failed to watch *v1.PersistentVolume: failed to list *v1.PersistentVolume: Get "https://192.168.18.15:6443/api/v1/persistentvolumes?limit=500&resourceVersion=0": dial tcp 192.168.18.15:6443: connect: network is unreachable
E0119 17:21:43.637908       1 reflector.go:138] k8s.io/client-go/informers/factory.go:134: Failed to watch *v1.Service: failed to list *v1.Service: Get "https://192.168.18.15:6443/api/v1/services?limit=500&resourceVersion=0": dial tcp 192.168.18.15:6443: connect: network is unreachable
E0119 17:21:45.453099       1 reflector.go:138] k8s.io/client-go/informers/factory.go:134: Failed to watch *v1.Pod: failed to list *v1.Pod: Get "https://192.168.18.15:6443/api/v1/pods?fieldSelector=status.phase%21%3DSucceeded%2Cstatus.phase%21%3DFailed&limit=500&resourceVersion=0": dial tcp 192.168.18.15:6443: connect: network is unreachable
E0119 17:21:45.650181       1 reflector.go:138] k8s.io/apiserver/pkg/server/dynamiccertificates/configmap_cafile_content.go:206: Failed to watch *v1.ConfigMap: failed to list *v1.ConfigMap: Get "https://192.168.18.15:6443/api/v1/namespaces/kube-system/configmaps?fieldSelector=metadata.name%3Dextension-apiserver-authentication&limit=500&resourceVersion=0": dial tcp 192.168.18.15:6443: connect: network is unreachable
E0119 17:21:46.981998       1 reflector.go:138] k8s.io/client-go/informers/factory.go:134: Failed to watch *v1.ReplicaSet: failed to list *v1.ReplicaSet: Get "https://192.168.18.15:6443/apis/apps/v1/replicasets?limit=500&resourceVersion=0": dial tcp 192.168.18.15:6443: connect: network is unreachable
E0119 17:21:52.151589       1 reflector.go:138] k8s.io/client-go/informers/factory.go:134: Failed to watch *v1.PersistentVolumeClaim: failed to list *v1.PersistentVolumeClaim: Get "https://192.168.18.15:6443/api/v1/persistentvolumeclaims?limit=500&resourceVersion=0": dial tcp 192.168.18.15:6443: connect: network is unreachable
E0119 17:21:55.824533       1 reflector.go:138] k8s.io/client-go/informers/factory.go:134: Failed to watch *v1.CSINode: failed to list *v1.CSINode: Get "https://192.168.18.15:6443/apis/storage.k8s.io/v1/csinodes?limit=500&resourceVersion=0": dial tcp 192.168.18.15:6443: connect: network is unreachable
E0119 17:21:59.126578       1 reflector.go:138] k8s.io/client-go/informers/factory.go:134: Failed to watch *v1.ReplicationController: failed to list *v1.ReplicationController: Get "https://192.168.18.15:6443/api/v1/replicationcontrollers?limit=500&resourceVersion=0": dial tcp 192.168.18.15:6443: connect: network is unreachable
E0119 17:22:04.326778       1 reflector.go:138] k8s.io/client-go/informers/factory.go:134: Failed to watch *v1.Node: failed to list *v1.Node: Get "https://192.168.18.15:6443/api/v1/nodes?limit=500&resourceVersion=0": dial tcp 192.168.18.15:6443: connect: network is unreachable
E0119 17:22:07.195279       1 reflector.go:138] k8s.io/client-go/informers/factory.go:134: Failed to watch *v1.StatefulSet: failed to list *v1.StatefulSet: Get "https://192.168.18.15:6443/apis/apps/v1/statefulsets?limit=500&resourceVersion=0": dial tcp 192.168.18.15:6443: connect: network is unreachable
E0119 17:22:08.487886       1 reflector.go:138] k8s.io/client-go/informers/factory.go:134: Failed to watch *v1.StorageClass: failed to list *v1.StorageClass: Get "https://192.168.18.15:6443/apis/storage.k8s.io/v1/storageclasses?limit=500&resourceVersion=0": dial tcp 192.168.18.15:6443: connect: network is unreachable
E0119 17:22:16.297515       1 reflector.go:138] k8s.io/client-go/informers/factory.go:134: Failed to watch *v1.Pod: failed to list *v1.Pod: Get "https://192.168.18.15:6443/api/v1/pods?fieldSelector=status.phase%21%3DSucceeded%2Cstatus.phase%21%3DFailed&limit=500&resourceVersion=0": dial tcp 192.168.18.15:6443: connect: network is unreachable
E0119 17:22:26.088227       1 reflector.go:138] k8s.io/client-go/informers/factory.go:134: Failed to watch *v1.CSINode: failed to list *v1.CSINode: Get "https://192.168.18.15:6443/apis/storage.k8s.io/v1/csinodes?limit=500&resourceVersion=0": dial tcp 192.168.18.15:6443: connect: network is unreachable
E0119 17:22:29.402834       1 reflector.go:138] k8s.io/client-go/informers/factory.go:134: Failed to watch *v1.PersistentVolumeClaim: failed to list *v1.PersistentVolumeClaim: Get "https://192.168.18.15:6443/api/v1/persistentvolumeclaims?limit=500&resourceVersion=0": dial tcp 192.168.18.15:6443: connect: network is unreachable
E0120 04:59:00.660266       1 reflector.go:138] k8s.io/client-go/informers/factory.go:134: Failed to watch *v1.PersistentVolumeClaim: failed to list *v1.PersistentVolumeClaim: Get "https://192.168.18.15:6443/api/v1/persistentvolumeclaims?limit=500&resourceVersion=0": dial tcp 192.168.18.15:6443: connect: network is unreachable
E0120 04:59:00.660542       1 reflector.go:138] k8s.io/apiserver/pkg/server/dynamiccertificates/configmap_cafile_content.go:206: Failed to watch *v1.ConfigMap: failed to list *v1.ConfigMap: Get "https://192.168.18.15:6443/api/v1/namespaces/kube-system/configmaps?fieldSelector=metadata.name%3Dextension-apiserver-authentication&limit=500&resourceVersion=0": dial tcp 192.168.18.15:6443: connect: network is unreachable
E0120 04:59:00.660679       1 reflector.go:138] k8s.io/client-go/informers/factory.go:134: Failed to watch *v1.StorageClass: failed to list *v1.StorageClass: Get "https://192.168.18.15:6443/apis/storage.k8s.io/v1/storageclasses?limit=500&resourceVersion=0": dial tcp 192.168.18.15:6443: connect: network is unreachable
E0120 04:59:00.660881       1 reflector.go:138] k8s.io/client-go/informers/factory.go:134: Failed to watch *v1.CSINode: failed to list *v1.CSINode: Get "https://192.168.18.15:6443/apis/storage.k8s.io/v1/csinodes?limit=500&resourceVersion=0": dial tcp 192.168.18.15:6443: connect: network is unreachable
E0120 04:59:00.794316       1 reflector.go:138] k8s.io/client-go/informers/factory.go:134: Failed to watch *v1.Pod: failed to list *v1.Pod: Get "https://192.168.18.15:6443/api/v1/pods?fieldSelector=status.phase%21%3DSucceeded%2Cstatus.phase%21%3DFailed&limit=500&resourceVersion=0": dial tcp 192.168.18.15:6443: connect: network is unreachable
E0120 04:59:00.794944       1 reflector.go:138] k8s.io/client-go/informers/factory.go:134: Failed to watch *v1.ReplicaSet: failed to list *v1.ReplicaSet: Get "https://192.168.18.15:6443/apis/apps/v1/replicasets?limit=500&resourceVersion=0": dial tcp 192.168.18.15:6443: connect: network is unreachable
E0120 04:59:00.795322       1 reflector.go:138] k8s.io/client-go/informers/factory.go:134: Failed to watch *v1beta1.PodDisruptionBudget: failed to list *v1beta1.PodDisruptionBudget: Get "https://192.168.18.15:6443/apis/policy/v1beta1/poddisruptionbudgets?limit=500&resourceVersion=0": dial tcp 192.168.18.15:6443: connect: network is unreachable
E0120 04:59:00.795551       1 reflector.go:138] k8s.io/client-go/informers/factory.go:134: Failed to watch *v1.Node: failed to list *v1.Node: Get "https://192.168.18.15:6443/api/v1/nodes?limit=500&resourceVersion=0": dial tcp 192.168.18.15:6443: connect: network is unreachable
E0120 04:59:00.799742       1 reflector.go:138] k8s.io/client-go/informers/factory.go:134: Failed to watch *v1.ReplicationController: failed to list *v1.ReplicationController: Get "https://192.168.18.15:6443/api/v1/replicationcontrollers?limit=500&resourceVersion=0": dial tcp 192.168.18.15:6443: connect: network is unreachable
E0120 04:59:00.800099       1 reflector.go:138] k8s.io/client-go/informers/factory.go:134: Failed to watch *v1.PersistentVolume: failed to list *v1.PersistentVolume: Get "https://192.168.18.15:6443/api/v1/persistentvolumes?limit=500&resourceVersion=0": dial tcp 192.168.18.15:6443: connect: network is unreachable
E0120 04:59:00.800422       1 reflector.go:138] k8s.io/client-go/informers/factory.go:134: Failed to watch *v1.StatefulSet: failed to list *v1.StatefulSet: Get "https://192.168.18.15:6443/apis/apps/v1/statefulsets?limit=500&resourceVersion=0": dial tcp 192.168.18.15:6443: connect: network is unreachable
E0120 04:59:00.800729       1 reflector.go:138] k8s.io/client-go/informers/factory.go:134: Failed to watch *v1.Service: failed to list *v1.Service: Get "https://192.168.18.15:6443/api/v1/services?limit=500&resourceVersion=0": dial tcp 192.168.18.15:6443: connect: network is unreachable
E0120 04:59:01.631819       1 reflector.go:138] k8s.io/apiserver/pkg/server/dynamiccertificates/configmap_cafile_content.go:206: Failed to watch *v1.ConfigMap: failed to list *v1.ConfigMap: Get "https://192.168.18.15:6443/api/v1/namespaces/kube-system/configmaps?fieldSelector=metadata.name%3Dextension-apiserver-authentication&limit=500&resourceVersion=0": dial tcp 192.168.18.15:6443: connect: network is unreachable
E0120 04:59:01.669605       1 reflector.go:138] k8s.io/client-go/informers/factory.go:134: Failed to watch *v1beta1.PodDisruptionBudget: failed to list *v1beta1.PodDisruptionBudget: Get "https://192.168.18.15:6443/apis/policy/v1beta1/poddisruptionbudgets?limit=500&resourceVersion=0": dial tcp 192.168.18.15:6443: connect: network is unreachable
E0120 04:59:01.717825       1 reflector.go:138] k8s.io/client-go/informers/factory.go:134: Failed to watch *v1.StorageClass: failed to list *v1.StorageClass: Get "https://192.168.18.15:6443/apis/storage.k8s.io/v1/storageclasses?limit=500&resourceVersion=0": dial tcp 192.168.18.15:6443: connect: network is unreachable
E0120 04:59:01.779959       1 reflector.go:138] k8s.io/client-go/informers/factory.go:134: Failed to watch *v1.StatefulSet: failed to list *v1.StatefulSet: Get "https://192.168.18.15:6443/apis/apps/v1/statefulsets?limit=500&resourceVersion=0": dial tcp 192.168.18.15:6443: connect: network is unreachable
E0120 04:59:01.806198       1 reflector.go:138] k8s.io/client-go/informers/factory.go:134: Failed to watch *v1.PersistentVolume: failed to list *v1.PersistentVolume: Get "https://192.168.18.15:6443/api/v1/persistentvolumes?limit=500&resourceVersion=0": dial tcp 192.168.18.15:6443: connect: network is unreachable
E0120 04:59:01.806524       1 reflector.go:138] k8s.io/client-go/informers/factory.go:134: Failed to watch *v1.CSINode: failed to list *v1.CSINode: Get "https://192.168.18.15:6443/apis/storage.k8s.io/v1/csinodes?limit=500&resourceVersion=0": dial tcp 192.168.18.15:6443: connect: network is unreachable
E0120 04:59:01.812570       1 reflector.go:138] k8s.io/client-go/informers/factory.go:134: Failed to watch *v1.Node: failed to list *v1.Node: Get "https://192.168.18.15:6443/api/v1/nodes?limit=500&resourceVersion=0": dial tcp 192.168.18.15:6443: connect: network is unreachable
E0120 04:59:01.866313       1 reflector.go:138] k8s.io/client-go/informers/factory.go:134: Failed to watch *v1.ReplicationController: failed to list *v1.ReplicationController: Get "https://192.168.18.15:6443/api/v1/replicationcontrollers?limit=500&resourceVersion=0": dial tcp 192.168.18.15:6443: connect: network is unreachable
E0120 04:59:01.877445       1 reflector.go:138] k8s.io/client-go/informers/factory.go:134: Failed to watch *v1.Service: failed to list *v1.Service: Get "https://192.168.18.15:6443/api/v1/services?limit=500&resourceVersion=0": dial tcp 192.168.18.15:6443: connect: network is unreachable
E0120 04:59:01.918989       1 reflector.go:138] k8s.io/client-go/informers/factory.go:134: Failed to watch *v1.ReplicaSet: failed to list *v1.ReplicaSet: Get "https://192.168.18.15:6443/apis/apps/v1/replicasets?limit=500&resourceVersion=0": dial tcp 192.168.18.15:6443: connect: network is unreachable
E0120 04:59:01.983466       1 reflector.go:138] k8s.io/client-go/informers/factory.go:134: Failed to watch *v1.PersistentVolumeClaim: failed to list *v1.PersistentVolumeClaim: Get "https://192.168.18.15:6443/api/v1/persistentvolumeclaims?limit=500&resourceVersion=0": dial tcp 192.168.18.15:6443: connect: network is unreachable
E0120 04:59:02.259326       1 reflector.go:138] k8s.io/client-go/informers/factory.go:134: Failed to watch *v1.Pod: failed to list *v1.Pod: Get "https://192.168.18.15:6443/api/v1/pods?fieldSelector=status.phase%21%3DSucceeded%2Cstatus.phase%21%3DFailed&limit=500&resourceVersion=0": dial tcp 192.168.18.15:6443: connect: network is unreachable
E0120 04:59:03.342799       1 reflector.go:138] k8s.io/client-go/informers/factory.go:134: Failed to watch *v1.StorageClass: failed to list *v1.StorageClass: Get "https://192.168.18.15:6443/apis/storage.k8s.io/v1/storageclasses?limit=500&resourceVersion=0": dial tcp 192.168.18.15:6443: connect: network is unreachable
E0120 04:59:03.619974       1 reflector.go:138] k8s.io/apiserver/pkg/server/dynamiccertificates/configmap_cafile_content.go:206: Failed to watch *v1.ConfigMap: failed to list *v1.ConfigMap: Get "https://192.168.18.15:6443/api/v1/namespaces/kube-system/configmaps?fieldSelector=metadata.name%3Dextension-apiserver-authentication&limit=500&resourceVersion=0": dial tcp 192.168.18.15:6443: connect: network is unreachable
E0120 04:59:03.740106       1 reflector.go:138] k8s.io/client-go/informers/factory.go:134: Failed to watch *v1.ReplicationController: failed to list *v1.ReplicationController: Get "https://192.168.18.15:6443/api/v1/replicationcontrollers?limit=500&resourceVersion=0": dial tcp 192.168.18.15:6443: connect: network is unreachable
E0120 04:59:03.740477       1 reflector.go:138] k8s.io/client-go/informers/factory.go:134: Failed to watch *v1beta1.PodDisruptionBudget: failed to list *v1beta1.PodDisruptionBudget: Get "https://192.168.18.15:6443/apis/policy/v1beta1/poddisruptionbudgets?limit=500&resourceVersion=0": dial tcp 192.168.18.15:6443: connect: network is unreachable
E0120 04:59:04.197358       1 reflector.go:138] k8s.io/client-go/informers/factory.go:134: Failed to watch *v1.CSINode: failed to list *v1.CSINode: Get "https://192.168.18.15:6443/apis/storage.k8s.io/v1/csinodes?limit=500&resourceVersion=0": dial tcp 192.168.18.15:6443: connect: network is unreachable
E0120 04:59:04.202881       1 reflector.go:138] k8s.io/client-go/informers/factory.go:134: Failed to watch *v1.Service: failed to list *v1.Service: Get "https://192.168.18.15:6443/api/v1/services?limit=500&resourceVersion=0": dial tcp 192.168.18.15:6443: connect: network is unreachable
E0120 04:59:04.215125       1 reflector.go:138] k8s.io/client-go/informers/factory.go:134: Failed to watch *v1.Pod: failed to list *v1.Pod: Get "https://192.168.18.15:6443/api/v1/pods?fieldSelector=status.phase%21%3DSucceeded%2Cstatus.phase%21%3DFailed&limit=500&resourceVersion=0": dial tcp 192.168.18.15:6443: connect: network is unreachable
E0120 04:59:04.328020       1 reflector.go:138] k8s.io/client-go/informers/factory.go:134: Failed to watch *v1.PersistentVolume: failed to list *v1.PersistentVolume: Get "https://192.168.18.15:6443/api/v1/persistentvolumes?limit=500&resourceVersion=0": dial tcp 192.168.18.15:6443: connect: network is unreachable
E0120 04:59:04.349680       1 reflector.go:138] k8s.io/client-go/informers/factory.go:134: Failed to watch *v1.ReplicaSet: failed to list *v1.ReplicaSet: Get "https://192.168.18.15:6443/apis/apps/v1/replicasets?limit=500&resourceVersion=0": dial tcp 192.168.18.15:6443: connect: network is unreachable
E0120 04:59:04.460854       1 reflector.go:138] k8s.io/client-go/informers/factory.go:134: Failed to watch *v1.StatefulSet: failed to list *v1.StatefulSet: Get "https://192.168.18.15:6443/apis/apps/v1/statefulsets?limit=500&resourceVersion=0": dial tcp 192.168.18.15:6443: connect: network is unreachable
E0120 04:59:04.491496       1 reflector.go:138] k8s.io/client-go/informers/factory.go:134: Failed to watch *v1.Node: failed to list *v1.Node: Get "https://192.168.18.15:6443/api/v1/nodes?limit=500&resourceVersion=0": dial tcp 192.168.18.15:6443: connect: network is unreachable
E0120 04:59:04.664730       1 reflector.go:138] k8s.io/client-go/informers/factory.go:134: Failed to watch *v1.PersistentVolumeClaim: failed to list *v1.PersistentVolumeClaim: Get "https://192.168.18.15:6443/api/v1/persistentvolumeclaims?limit=500&resourceVersion=0": dial tcp 192.168.18.15:6443: connect: network is unreachable
E0120 04:59:06.705867       1 reflector.go:138] k8s.io/client-go/informers/factory.go:134: Failed to watch *v1.StorageClass: failed to list *v1.StorageClass: Get "https://192.168.18.15:6443/apis/storage.k8s.io/v1/storageclasses?limit=500&resourceVersion=0": dial tcp 192.168.18.15:6443: connect: network is unreachable
E0120 04:59:07.291919       1 reflector.go:138] k8s.io/client-go/informers/factory.go:134: Failed to watch *v1beta1.PodDisruptionBudget: failed to list *v1beta1.PodDisruptionBudget: Get "https://192.168.18.15:6443/apis/policy/v1beta1/poddisruptionbudgets?limit=500&resourceVersion=0": dial tcp 192.168.18.15:6443: connect: network is unreachable
E0120 04:59:07.554727       1 reflector.go:138] k8s.io/client-go/informers/factory.go:134: Failed to watch *v1.ReplicationController: failed to list *v1.ReplicationController: Get "https://192.168.18.15:6443/api/v1/replicationcontrollers?limit=500&resourceVersion=0": dial tcp 192.168.18.15:6443: connect: network is unreachable
E0120 04:59:08.367599       1 reflector.go:138] k8s.io/client-go/informers/factory.go:134: Failed to watch *v1.PersistentVolumeClaim: failed to list *v1.PersistentVolumeClaim: Get "https://192.168.18.15:6443/api/v1/persistentvolumeclaims?limit=500&resourceVersion=0": dial tcp 192.168.18.15:6443: connect: network is unreachable
E0120 04:59:08.565792       1 reflector.go:138] k8s.io/client-go/informers/factory.go:134: Failed to watch *v1.Pod: failed to list *v1.Pod: Get "https://192.168.18.15:6443/api/v1/pods?fieldSelector=status.phase%21%3DSucceeded%2Cstatus.phase%21%3DFailed&limit=500&resourceVersion=0": dial tcp 192.168.18.15:6443: connect: network is unreachable
E0120 04:59:09.193342       1 reflector.go:138] k8s.io/client-go/informers/factory.go:134: Failed to watch *v1.Service: failed to list *v1.Service: Get "https://192.168.18.15:6443/api/v1/services?limit=500&resourceVersion=0": dial tcp 192.168.18.15:6443: connect: network is unreachable
E0120 04:59:09.340462       1 reflector.go:138] k8s.io/client-go/informers/factory.go:134: Failed to watch *v1.CSINode: failed to list *v1.CSINode: Get "https://192.168.18.15:6443/apis/storage.k8s.io/v1/csinodes?limit=500&resourceVersion=0": dial tcp 192.168.18.15:6443: connect: network is unreachable
E0120 04:59:09.391302       1 reflector.go:138] k8s.io/apiserver/pkg/server/dynamiccertificates/configmap_cafile_content.go:206: Failed to watch *v1.ConfigMap: failed to list *v1.ConfigMap: Get "https://192.168.18.15:6443/api/v1/namespaces/kube-system/configmaps?fieldSelector=metadata.name%3Dextension-apiserver-authentication&limit=500&resourceVersion=0": dial tcp 192.168.18.15:6443: connect: network is unreachable
E0120 04:59:09.698258       1 reflector.go:138] k8s.io/client-go/informers/factory.go:134: Failed to watch *v1.PersistentVolume: failed to list *v1.PersistentVolume: Get "https://192.168.18.15:6443/api/v1/persistentvolumes?limit=500&resourceVersion=0": dial tcp 192.168.18.15:6443: connect: network is unreachable
E0120 04:59:09.898596       1 reflector.go:138] k8s.io/client-go/informers/factory.go:134: Failed to watch *v1.StatefulSet: failed to list *v1.StatefulSet: Get "https://192.168.18.15:6443/apis/apps/v1/statefulsets?limit=500&resourceVersion=0": dial tcp 192.168.18.15:6443: connect: network is unreachable
E0120 04:59:10.464583       1 reflector.go:138] k8s.io/client-go/informers/factory.go:134: Failed to watch *v1.ReplicaSet: failed to list *v1.ReplicaSet: Get "https://192.168.18.15:6443/apis/apps/v1/replicasets?limit=500&resourceVersion=0": dial tcp 192.168.18.15:6443: connect: network is unreachable
E0120 04:59:10.741245       1 reflector.go:138] k8s.io/client-go/informers/factory.go:134: Failed to watch *v1.Node: failed to list *v1.Node: Get "https://192.168.18.15:6443/api/v1/nodes?limit=500&resourceVersion=0": dial tcp 192.168.18.15:6443: connect: network is unreachable
E0120 04:59:16.793760       1 reflector.go:138] k8s.io/client-go/informers/factory.go:134: Failed to watch *v1.ReplicationController: failed to list *v1.ReplicationController: Get "https://192.168.18.15:6443/api/v1/replicationcontrollers?limit=500&resourceVersion=0": dial tcp 192.168.18.15:6443: connect: network is unreachable
E0120 04:59:17.365539       1 reflector.go:138] k8s.io/client-go/informers/factory.go:134: Failed to watch *v1beta1.PodDisruptionBudget: failed to list *v1beta1.PodDisruptionBudget: Get "https://192.168.18.15:6443/apis/policy/v1beta1/poddisruptionbudgets?limit=500&resourceVersion=0": dial tcp 192.168.18.15:6443: connect: network is unreachable
E0120 04:59:18.973337       1 reflector.go:138] k8s.io/client-go/informers/factory.go:134: Failed to watch *v1.PersistentVolumeClaim: failed to list *v1.PersistentVolumeClaim: Get "https://192.168.18.15:6443/api/v1/persistentvolumeclaims?limit=500&resourceVersion=0": dial tcp 192.168.18.15:6443: connect: network is unreachable
E0120 04:59:19.039719       1 reflector.go:138] k8s.io/client-go/informers/factory.go:134: Failed to watch *v1.PersistentVolume: failed to list *v1.PersistentVolume: Get "https://192.168.18.15:6443/api/v1/persistentvolumes?limit=500&resourceVersion=0": dial tcp 192.168.18.15:6443: connect: network is unreachable
E0120 04:59:19.441270       1 reflector.go:138] k8s.io/client-go/informers/factory.go:134: Failed to watch *v1.StorageClass: failed to list *v1.StorageClass: Get "https://192.168.18.15:6443/apis/storage.k8s.io/v1/storageclasses?limit=500&resourceVersion=0": dial tcp 192.168.18.15:6443: connect: network is unreachable
E0120 04:59:19.601796       1 reflector.go:138] k8s.io/client-go/informers/factory.go:134: Failed to watch *v1.CSINode: failed to list *v1.CSINode: Get "https://192.168.18.15:6443/apis/storage.k8s.io/v1/csinodes?limit=500&resourceVersion=0": dial tcp 192.168.18.15:6443: connect: network is unreachable
E0120 04:59:19.915139       1 reflector.go:138] k8s.io/client-go/informers/factory.go:134: Failed to watch *v1.Service: failed to list *v1.Service: Get "https://192.168.18.15:6443/api/v1/services?limit=500&resourceVersion=0": dial tcp 192.168.18.15:6443: connect: network is unreachable
E0120 04:59:20.104724       1 reflector.go:138] k8s.io/client-go/informers/factory.go:134: Failed to watch *v1.StatefulSet: failed to list *v1.StatefulSet: Get "https://192.168.18.15:6443/apis/apps/v1/statefulsets?limit=500&resourceVersion=0": dial tcp 192.168.18.15:6443: connect: network is unreachable
E0120 04:59:20.104847       1 reflector.go:138] k8s.io/client-go/informers/factory.go:134: Failed to watch *v1.Pod: failed to list *v1.Pod: Get "https://192.168.18.15:6443/api/v1/pods?fieldSelector=status.phase%21%3DSucceeded%2Cstatus.phase%21%3DFailed&limit=500&resourceVersion=0": dial tcp 192.168.18.15:6443: connect: network is unreachable
E0120 04:59:21.382862       1 reflector.go:138] k8s.io/client-go/informers/factory.go:134: Failed to watch *v1.Node: failed to list *v1.Node: Get "https://192.168.18.15:6443/api/v1/nodes?limit=500&resourceVersion=0": dial tcp 192.168.18.15:6443: connect: network is unreachable
E0120 04:59:21.801067       1 reflector.go:138] k8s.io/apiserver/pkg/server/dynamiccertificates/configmap_cafile_content.go:206: Failed to watch *v1.ConfigMap: failed to list *v1.ConfigMap: Get "https://192.168.18.15:6443/api/v1/namespaces/kube-system/configmaps?fieldSelector=metadata.name%3Dextension-apiserver-authentication&limit=500&resourceVersion=0": dial tcp 192.168.18.15:6443: connect: network is unreachable
E0120 04:59:22.336526       1 reflector.go:138] k8s.io/client-go/informers/factory.go:134: Failed to watch *v1.ReplicaSet: failed to list *v1.ReplicaSet: Get "https://192.168.18.15:6443/apis/apps/v1/replicasets?limit=500&resourceVersion=0": dial tcp 192.168.18.15:6443: connect: network is unreachable
E0120 04:59:32.139132       1 reflector.go:138] k8s.io/client-go/informers/factory.go:134: Failed to watch *v1.PersistentVolume: failed to list *v1.PersistentVolume: Get "https://192.168.18.15:6443/api/v1/persistentvolumes?limit=500&resourceVersion=0": dial tcp 192.168.18.15:6443: connect: network is unreachable
E0120 04:59:33.791449       1 reflector.go:138] k8s.io/client-go/informers/factory.go:134: Failed to watch *v1.CSINode: failed to list *v1.CSINode: Get "https://192.168.18.15:6443/apis/storage.k8s.io/v1/csinodes?limit=500&resourceVersion=0": dial tcp 192.168.18.15:6443: connect: network is unreachable
E0120 04:59:34.363716       1 reflector.go:138] k8s.io/client-go/informers/factory.go:134: Failed to watch *v1beta1.PodDisruptionBudget: failed to list *v1beta1.PodDisruptionBudget: Get "https://192.168.18.15:6443/apis/policy/v1beta1/poddisruptionbudgets?limit=500&resourceVersion=0": dial tcp 192.168.18.15:6443: connect: network is unreachable
E0120 04:59:34.887750       1 reflector.go:138] k8s.io/client-go/informers/factory.go:134: Failed to watch *v1.Pod: failed to list *v1.Pod: Get "https://192.168.18.15:6443/api/v1/pods?fieldSelector=status.phase%21%3DSucceeded%2Cstatus.phase%21%3DFailed&limit=500&resourceVersion=0": dial tcp 192.168.18.15:6443: connect: network is unreachable
E0120 04:59:36.338888       1 reflector.go:138] k8s.io/client-go/informers/factory.go:134: Failed to watch *v1.PersistentVolumeClaim: failed to list *v1.PersistentVolumeClaim: Get "https://192.168.18.15:6443/api/v1/persistentvolumeclaims?limit=500&resourceVersion=0": dial tcp 192.168.18.15:6443: connect: network is unreachable
E0120 04:59:36.370022       1 reflector.go:138] k8s.io/apiserver/pkg/server/dynamiccertificates/configmap_cafile_content.go:206: Failed to watch *v1.ConfigMap: failed to list *v1.ConfigMap: Get "https://192.168.18.15:6443/api/v1/namespaces/kube-system/configmaps?fieldSelector=metadata.name%3Dextension-apiserver-authentication&limit=500&resourceVersion=0": dial tcp 192.168.18.15:6443: connect: network is unreachable
E0120 04:59:39.606900       1 reflector.go:138] k8s.io/client-go/informers/factory.go:134: Failed to watch *v1.StatefulSet: failed to list *v1.StatefulSet: Get "https://192.168.18.15:6443/apis/apps/v1/statefulsets?limit=500&resourceVersion=0": dial tcp 192.168.18.15:6443: connect: network is unreachable
E0120 04:59:41.696198       1 reflector.go:138] k8s.io/client-go/informers/factory.go:134: Failed to watch *v1.ReplicationController: failed to list *v1.ReplicationController: Get "https://192.168.18.15:6443/api/v1/replicationcontrollers?limit=500&resourceVersion=0": dial tcp 192.168.18.15:6443: connect: network is unreachable
E0120 04:59:41.793942       1 reflector.go:138] k8s.io/client-go/informers/factory.go:134: Failed to watch *v1.Node: failed to list *v1.Node: Get "https://192.168.18.15:6443/api/v1/nodes?limit=500&resourceVersion=0": dial tcp 192.168.18.15:6443: connect: network is unreachable
E0120 04:59:42.918414       1 reflector.go:138] k8s.io/client-go/informers/factory.go:134: Failed to watch *v1.StorageClass: failed to list *v1.StorageClass: Get "https://192.168.18.15:6443/apis/storage.k8s.io/v1/storageclasses?limit=500&resourceVersion=0": dial tcp 192.168.18.15:6443: connect: network is unreachable
E0120 04:59:44.188410       1 reflector.go:138] k8s.io/client-go/informers/factory.go:134: Failed to watch *v1.Service: failed to list *v1.Service: Get "https://192.168.18.15:6443/api/v1/services?limit=500&resourceVersion=0": dial tcp 192.168.18.15:6443: connect: network is unreachable
E0120 04:59:47.105061       1 reflector.go:138] k8s.io/client-go/informers/factory.go:134: Failed to watch *v1.ReplicaSet: failed to list *v1.ReplicaSet: Get "https://192.168.18.15:6443/apis/apps/v1/replicasets?limit=500&resourceVersion=0": dial tcp 192.168.18.15:6443: connect: network is unreachable
E0120 04:59:58.047111       1 reflector.go:138] k8s.io/client-go/informers/factory.go:134: Failed to watch *v1.PersistentVolume: failed to list *v1.PersistentVolume: Get "https://192.168.18.15:6443/api/v1/persistentvolumes?limit=500&resourceVersion=0": dial tcp 192.168.18.15:6443: connect: network is unreachable
I0120 05:00:15.744557       1 shared_informer.go:247] Caches are synced for client-ca::kube-system::extension-apiserver-authentication::client-ca-file
I0120 05:00:40.044748       1 leaderelection.go:243] attempting to acquire leader lease kube-system/kube-scheduler...
I0120 05:00:56.790000       1 leaderelection.go:253] successfully acquired lease kube-system/kube-scheduler
[root@master ~]#
[root@master ~]# netstat -tulpn | grep 6443
tcp6       0      0 :::6443                 :::*                    LISTEN      28951/kube-apiserve
[root@master ~]#
